Reaktoro/Core/Phases.hpp

```diff
@@ -1,387 +1,421 @@
-// Reaktoro is a unified framework for modeling chemically reactive systems.

```

Comment:
I don't understand why this file is marked here as completely different. It's not possible to see the minor changes that occurred in it.

Reply:
Honestly, I don't know either.